src/models/index.js

```diff
@@ -18,7 +18,9 @@ Model.prototype.pull = function (req, callback) {
 }
 
 function createKey (req) {
-  let key = req.url.split('/')[1]
+  let key = (req.params.layer)
+    ? req.url.split('/')[1] + '_' + req.params.layer
+    : req.url.split('/')[1]
   if (req.params.host) key = [key, req.params.host].join('::')
   if (req.params.id) key = [key, req.params.id].join('::')
   return key
```

Comment:
I would suggest just following the pattern already used for `host` and `id`, i.e.,
`if (req.params.layer) key = [key, req.params.layer].join('::')`

Also, append the layer fragment of the key after `host` and `id` since that would make the key composition go from general to specific in left-to-right order.